tee-worker/ts-tests/bulk_identity.test.ts

```diff
@@ -7,66 +7,65 @@ import {
     assertIdentityCreated,
     assertIdentityRemoved,
     assertIdentityVerified,
+    assertInitialIDGraphCreated,
 } from './common/utils';
 import { KeyringPair } from '@polkadot/keyring/types';
 import { ethers } from 'ethers';
 import { LitentryIdentity, LitentryValidationData } from './common/type-definitions';
 import { handleIdentityEvents } from './common/utils';
-import { assert } from 'chai';
-import { listenEvent, multiAccountTxSender } from './common/transactions';
-import { u8aToHex } from '@polkadot/util';
-import { ApiTypes, SubmittableExtrinsic } from '@polkadot/api/types';
+import { multiAccountTxSender } from './common/transactions';
+import { SubmittableResult } from '@polkadot/api';
 
 //Explain how to use this test, which has two important parameters:
 //1.The "number" parameter in describeLitentry represents the number of accounts generated, including Substrate wallets and Ethereum wallets.If you want to use a large number of accounts for testing, you can modify this parameter.
 //2.Each time the test code is executed, new wallet account will be used.
 
-describeLitentry('multiple accounts test', 10, async (context) => {
+describeLitentry('multiple accounts test', 2, async (context) => {
     const aesKey = '0x22fc82db5b606998ad45099b7978b5b4f9dd4ea6017e57370ac56141caaabd12';
-    var substraetSigners: KeyringPair[] = [];
+    var substrateSigners: KeyringPair[] = [];
     var ethereumSigners: ethers.Wallet[] = [];
     var web3Validations: LitentryValidationData[] = [];
     var identities: LitentryIdentity[] = [];
     step('setup signers', async () => {
-        substraetSigners = context.web3Signers.map((web3Signer) => {
+        substrateSigners = context.web3Signers.map((web3Signer) => {
             return web3Signer.substrateWallet;
         });
         ethereumSigners = context.web3Signers.map((web3Signer) => {
             return web3Signer.ethereumWallet;
         });
     });
     step('send test token to each account', async () => {
-        const txs: SubmittableExtrinsic<ApiTypes>[] = [];
-        for (let i = 0; i < substraetSigners.length; i++) {
+        const txs: any = [];
```

Comment:
Must we? :sadpanda:

Reply:
I modified this type [here](https://github.com/litentry/litentry-parachain/pull/1703/files#diff-e9b9bfd1cf7ea674b2a3a4c883e38a401b6a06c08f90e00d69d60f6abbdd4b08R41).